spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/controller/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Spring Cloud Data REST & UI Controllers.
```

Comment:
"Spring Cloud Data *Admin* REST & UI Controllers." (or we can probably leave out "REST & UI").